### PR TITLE
Add green background according to beautiful design

### PR DESCRIPTION
**Pull request by nfilzi**
Green is beautiful!

### CODE REVIEW

**Comment by aristideriou** [comment]
It's superb but I'd prefer pink!

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 <style>
   .form-wrapper {
-    background-color: white;
```
Comment (aristideriou) [inline app/views/devise/registrations/new.html.erb]: Yeah, awesome !

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
 </style>
 
 <div class="container">
-  <div class="form-wrapper form-wrapper--signup" style="background-color: white; padding: 1em">
+  <div class="form-wrapper form-wrapper--signup">
 <h2>Sign up</h2>
 
 <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```
Comment (nfilzi) [inline app/views/devise/registrations/new.html.erb]: ozifjzoeifjozfjoziefjeozifjeoz

Comment (nfilzi) [reply app/views/devise/registrations/new.html.erb]: ejpfzjpzj

Comment (aristideriou) [reply app/views/devise/registrations/new.html.erb]: ¯\_(ツ)_/¯

**Pull request merged**